x-pack/plugin/sql/qa/src/main/resources/single-node-only/command-sys.csv-spec

```diff
@@ -0,0 +1,120 @@
+x-pack_plugin_sql_qa_single-node_integTestCluster  |null           |test\_emp      |gender            |12             |KEYWORD        |32766          |2147483647     |null           |null           |1              |null           |null           |12             |0               |2147483647       |6               |YES            |null           |null           |null           |null            |NO              |NO                
+x-pack_plugin_sql_qa_single-node_integTestCluster  |null           |test\_emp      |hire_date         |93             |DATETIME       |24             |8              |null           |null           |1              |null           |null           |9              |3               |null             |7               |YES            |null           |null           |null           |null            |NO              |NO                
+x-pack_plugin_sql_qa_single-node_integTestCluster  |null           |test\_emp      |languages         |-6             |BYTE           |5              |1              |null           |10             |1              |null           |null           |-6             |0               |null             |8               |YES            |null           |null           |null           |null            |NO              |NO                
+x-pack_plugin_sql_qa_single-node_integTestCluster  |null           |test\_emp      |last_name         |12             |TEXT           |2147483647     |2147483647     |null           |null           |1              |null           |null           |12             |0               |2147483647       |9               |YES            |null           |null           |null           |null            |NO              |NO                
+x-pack_plugin_sql_qa_single-node_integTestCluster  |null           |test\_emp      |last_name.keyword |12             |KEYWORD        |32766          |2147483647     |null           |null           |1              |null           |null           |12             |0               |2147483647       |10              |YES            |null           |null           |null           |null            |NO              |NO                
+x-pack_plugin_sql_qa_single-node_integTestCluster  |null           |test\_emp      |salary            |4              |INTEGER        |11             |4              |null           |10             |1              |null           |null           |4              |0               |null             |11              |YES            |null           |null           |null           |null            |NO              |NO                
+;
+
+sysColumnsWithTableLikeNoEscape
+SYS COLUMNS TABLE LIKE 'test_emp';
+
+// since there's no escaping test_emp means test*emp which matches also test_alias_emp
```

Comment:
`test_emp` should match `test`[one character]`emp` when there is no escape character, no?

Reply:
Unfortunately due to the current field caps features, we can't figure out the source index.